apps/site/lib/site_web/controllers/project_controller.ex

```diff
@@ -97,7 +95,7 @@ defmodule SiteWeb.ProjectController do
 
     "/projects"
     |> Path.join(project_alias)
-    |> get_page_fn.(preview_opts(conn.query_params))
+    |> get_page_fn.(conn.query_params)
```

Comment:
It's fine as a quick fix, but this seems contra to the separation-of-concerns idea discussed in #347 of having controllers be solely responsible for extracting and parsing query params. @amaisano would refactoring this be included in the new Asana task you created?

Reply:
Yes - this will be refactored to use the new `preview_opts` shape with another PR.